File: backend/app/tests/pcl5/schemas/pcl5_test.py
from app.schemas.test_base import TestBase
from app.tests.pcl5.schemas.pcl5_answer import PCL5Answer
from app.tests.pcl5.schemas.pcl5_criteria import PCL5Criteria
from app.tests.pcl5.schemas.pcl5_question import PCL5Question


class PCL5Test(TestBase):
    criterion: list[PCL5Criteria]
    answers: list[PCL5Answer]
    questions: list[PCL5Question]

File: frontend/src/features/tests/PCL5Test/types/PCL5Test.ts
import TestBase from "@/types/models/TestBase";
import PCL5Criteria from "@/features/tests/PCL5Test/types/PCL5Criteria";
import PCL5Answer from "@/features/tests/PCL5Test/types/PCL5Answer";
import PCL5Question from "@/features/tests/PCL5Test/types/PCL5Question";

export interface PCL5Test extends TestBase {
    criterion: PCL5Criteria[];
    answers: PCL5Answer[];
    questions: PCL5Question[];
}